app/src/main/java/uk/gov/android/securestore/crypto/HybridCryptoManager.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.android.securestore.crypto
+
+import uk.gov.android.securestore.AccessControlLevel
+import uk.gov.android.securestore.crypto.limitedmanager.LimitedCryptoManager
+
+/**
+ * Interface to handle encryption and decryption of [String] data
+ */
+interface HybridCryptoManager : LimitedCryptoManager {
+    /**
+     *Init the [AccessControlLevel] and alias, this must be done before using the CryptoManager
+     */
+    fun init(alias: String, acl: AccessControlLevel)
```

Comment:
Why use an `init` method and not the constructor for these values?
What happens if you call `init` multiple times between calling encrypt and decrypt?
It kinda looks like you are setting up a state machine with these methods, but I don't actually see that

Reply:
Alex has explained this